### DISCUSSION TITLE
Advanced Queue Settings

### DISCUSSION THREAD

**Discussion by ruizhannia** [nextgenhealthcare/connect#5324] [Q&A]
I am looking for some information on the Advanced Queue Settings, specifically on the 'retry interval'. I have an outbound feed that is set-up with 15000ms retry interval. Currently, I am seeing some messages that have had to retry to send, my question is, will the message retry to send in 15000ms or does it try to send for 15000ms and then re-attempts? If it helps, my send timeout is set to 15000ms as well. 
If I wanted to decrease the retry, do I increase my send timeout or my retry interval?

**Comment by mjtrac** [comment]
On a test send (to an inaccessible destination) with a Response Timeout of five seconds and one retry, with ten seconds between retries, the dashboard shows fifteen seconds between initial attempt and final failure, so it looks like the second attempt starts ten seconds in and fails after five more seconds.

On a second test send with a Response Timeout of seven seconds and two retries, with ten seconds between retries, the dashboard shows  26 seconds, so the three attempts start at zero, ten and 20 seconds, and the final failed retry ends after seven more seconds.

So, each time the message is sent, the system waits for the time you specify in Response Timeout for a response.  If it doesn't get a response, it sends again after the "retry interval" specified has elapsed from the start of the prior send.
